e2e-spock-geb/Jenkinsfile.template

```diff
@@ -42,8 +42,14 @@ def stageTest(def context) {
     springBootEnv = 'dev'
   }
 
+  def openShiftPublicHost
+  node {
+    def stableUrl = env.SONAR_SERVER_URL
```

Comment:
@metmajer  - we have to revisit this ... in lieu of @michaelsauter  - promote to P work ...

Reply:
oc create route edge my-route --service=frontend --port=80
oc get route my-route -o jsonpath='{.spec.host}'

remove "my-route" and "context.project" and done :)

oc delete route my-route

Reply:
@metmajer  . here we go

```
    def routeName = "test-route"
    sh ("oc create route edge ${routeName} --service=dummy --port=80 | true")
    def routeUrl = sh (script: "oc get route ${routeName} -o jsonpath='{.spec.host}'", returnStdout : true)
    def routePrefixLength = "${routeName}-${context.targetProject}".length() + 1
    openShiftPublicHost = routeUrl.substring(routePrefixLength)
    echo "oc domain: ${openShiftPublicHost}"
    sh ("oc delete route ${routeName} | true")
```

Reply:
I would prefer if this would be handled by @michaelsauter and the team in the context of their work.

Reply:
@metmajer . why? ...